src/x/xGetNoiseLevel.js

```diff
@@ -1,5 +1,5 @@
 import erfcinv from 'compute-erfcinv';
```

Comment:
Should we rename xGetNoiseLevel to something more specific ? Seems we will have many noise algorithm and adding a 'kind' would force to load all the dependencies of all the noise algorithm.

Reply:
the algorithm name is san-plot, what do you think about this name?

Reply:
xNoiseSanPlot